Pages/homePage.js

```diff
@@ -0,0 +1,44 @@
+class HomePage {
+    constructor() {
+        this.title= "HomePage";
+    }
+    get loginButton (){
+        return browser.driver.findElement(by.className("header-text"));
+    }
+    get fashionCategory(){
+        return browser.driver.findElement(by.css("div.category-item a"));
+    }
+
+    get brownJacket(){
+        return browser.actions().mouseMove(element(by.css("div.product-preview div a img")));
+    }
+    get bidField (){
+        return browser.driver.findElement(by.className("bid-input"));
+    }
+    get placeBidButton (){
```

Comment:
not on homepage

Reply:
review which elements are not on homepage, add new page object for them

Reply:
I reviewed which element belongs to which page(class).